.github/workflows/coverage.yml

```
@@ -19,6 +19,7 @@ jobs:
       # Configuration type to build.  For documentation on how build matrices work, see
       # https://docs.github.com/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
       BUILD_CONFIGURATION: ${{matrix.configurations}}
+      REPO_NAME: ${{ github.repository.split('/')[-1] }}
```

actionlint (1.7.4)

parser did not reach end of input after parsing the expression. 6 remaining token(s) in the input: "(", "STRING", ")", "[", "INTEGER", "]"

(expression)